account/identity.go

```diff
@@ -12,20 +13,62 @@ import (
 	"golang.org/x/net/context"
 )
 
-// Identity ddenDescribes a unique Person with the ALM
+const (
+	// KeycloakIDP is the name of Keycloak Identity Provider
+	KeycloakIDP string = "kc"
+	// GitHubIDP is the name of GitHub Identity Provider
+	GitHubIDP string = "gh"
```

Comment:
Yeah.. URLs may be better. Let's address it when we improving our remote WI import stuff. I will just remove GitHubIDP const for now since we do not use it now (we currently don't create identities when importing remote WIs).